drivers/sdhc/Kconfig.rcar

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config MMC_RCAR
+	bool "Renesas Rcar MMC driver"
+	select SDHC_SUPPORTS_NATIVE_MODE
+	select REGULATOR
+	select GPIO
+	select SDHC_SUPPORTS_UHS if SDMMC_STACK
+	default n
```

Comment:
This should be `default y`, with `depends on DT_HAS_RENESAS_RCAR_MMC_ENABLED`, see here: https://github.com/zephyrproject-rtos/zephyr/blob/18636af6d6658cf66e5f055901ffe2515116d329/drivers/sdhc/Kconfig.mcux_sdif#L7